common/tls/src/main/java/module-info.java

```diff
@@ -18,14 +18,16 @@
  * TLS configuration for client and server.
  */
 module io.helidon.common.tls {
+    requires static io.helidon.builder.api;
```

Comment:
This dependency (builder.api) should not be static (not sure why it was, it is a bug)